mix.exs

```diff
@@ -36,6 +36,7 @@ defmodule Honeybadger.Mixfile do
            notice_filter: Honeybadger.NoticeFilter.Default,
            filter: Honeybadger.Filter.Default,
            filter_keys: [:password, :credit_card],
+           filter_args: false,
```

Comment:
If this is an opt-in feature, we should default to `filter_args: true`. @joshuap thoughts?

Reply:
Yeah, I'm thinking we want to disable this feature by default. We discussed making it opt-in in https://github.com/honeybadger-io/honeybadger-elixir/issues/78#issuecomment-342577068. Does that make sense, @sorentwo?

Reply:
I've reversed the default, it is now `true`, making it opt-in like we discussed 👍